src/Idris/AbsSyntaxTree.hs

```diff
@@ -1655,7 +1655,7 @@ prettyName
   -> Doc OutputAnnotation
 prettyName infixParen showNS bnd n
     | (MN _ s) <- n, isPrefixOf "_" $ T.unpack s = text "_"
-    | (UN n') <- n, isPrefixOf "_" $ T.unpack n' = text "_"
+    | (UN n') <- n, T.unpack n' == "_" = text "_"
```

Comment:
Just FYI for the code reviewer, I am bit unsure if this change breaks something else also everything seemed to work.
If this change was not made, then it would have only displayed "_" for names starting with "_"